components/server/src/code-sync/code-sync-service.ts

```diff
@@ -24,7 +24,7 @@ import { Config } from '../config';
 import { CachingBlobServiceClientProvider } from '@gitpod/content-service/lib/sugar';
 
 // By default: 5 kind of resources * 20 revs * 1Mb = 100Mb max in the content service for user data.
-const defautltRevLimit = 20;
+const defaultRevLimit = 2;
```

Comment:
revert this :smile:

Reply:
It's only for debug, I'll revert this after review ok https://github.com/gitpod-io/gitpod/pull/8644/commits/6bd58d59ed5eb15ecf676e77939622748e252780